src/bundler/bundle_v2.zig

```diff
@@ -7450,17 +7450,31 @@ const LinkerContext = struct {
 
                     const r = lex.rangeOfIdentifier(source, named_import.alias_loc orelse Logger.Loc{});
 
-                    // if (result.name_loc.start != 0)
-                    // TODO: better error
-                    c.log.addRangeErrorFmt(
-                        source,
-                        r,
-                        c.allocator,
-                        "Ambiguous import: {s}",
-                        .{
-                            named_import.alias.?,
-                        },
-                    ) catch unreachable;
+                    // TODO: log locations of the ambiguous exports
+
+                    const symbol: *Symbol = c.graph.symbols.get(import_ref).?;
+                    if (symbol.import_item_status == .generated) {
+                        symbol.import_item_status = .missing;
+                        c.log.addRangeWarningFmt(
+                            source,
+                            r,
+                            c.allocator,
+                            "Import \"{s}\" will always be undefined because there are multiple matching exports",
```

Comment:
non-blocking:

Eventually we need to fix quoting everywhere in logs. We need a formatter that quotes & escapes quotes in error messages